modules/gapi/src/backends/fluid/gfluidcore_func.simd.hpp

```diff
@@ -1568,6 +1576,33 @@ ABSDIFFC_SIMD(float)
 
 #undef ABSDIFFC_SIMD
 
+//-------------------------
+//
+// Fluid kernels: Split3
+//
+//-------------------------
+
+#define SPLIT3_SIMD(SRC, DST)                                       \
+int split3_simd(const SRC in[], DST out1[], DST out2[],             \
```

Comment:
It makes no sense to make this function a macro for this kernel. Make this function a normal function.